chain/bitcoind_client.go

```diff
@@ -945,13 +945,6 @@ func (c *BitcoindClient) rescan(start chainhash.Hash) error {
 	}
 	headers.PushBack(previousHeader)
 
-	// Queue a RescanFinished notification to the caller with the last block
-	// processed throughout the rescan once done.
-	defer c.onRescanFinished(
-		previousHash, previousHeader.Height,
-		time.Unix(previousHeader.Time, 0),
```

Comment:
Nice catch! `previousHash` or `previousHeader` were also never even updated, so it's not as though they would've eventually pointed to the most up to date versions.